src/python/pants/core/goals/test.py

```diff
@@ -105,9 +111,10 @@ def skip(address: Address, output_setting: ShowOutput) -> TestResult:
         )
 
     @staticmethod
-    def skip_batch(
+    def no_tests_found_in_batch(
```

Comment:
This isn't called anywhere, should we just delete it?

Reply:
I'd be ok with deleting it - I only added it to keep 1:1 APIs for batched vs. unbatched